src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.component.html

```diff
@@ -181,7 +181,88 @@ <h4 *ngIf="isImport" id="jhi-modeling-exercise-heading-import" jhiTranslate="art
                     name="automaticAssessmentEnabled"
                     [disabled]="isImport"
                 />
-                <label class="form-check-label custom-control-label" for="automatic_assessment_enabled" jhiTranslate="artemisApp.textExercise.automaticAssessmentEnabled"></label>
+                <label
+                    class="form-check-label custom-control-label"
+                    for="automatic_assessment_enabled"
+                    jhiTranslate="artemisApp.modelingExercise.automaticAssessmentEnabled"
+                ></label>
+            </div>
+        </div>
+        <div class="form-group">
+            <div class="row">
+                <label jhiTranslate="artemisApp.exercise.continuousPlagiarismControl">Continuous Plagiarism Control</label>
+                <div class="plagiarism-option">
+                    <input
+                        type="checkbox"
+                        id="continuous_plagiarism_control_enabled"
+                        class="plagiarism-option-checkbox form-check-input"
+                        name="modelingExercise.continuousPlagiarismControlEnabled"
+                        #modelingExercise.continuousPlagiarismControlEnabled="ngModel"
+                        [(ngModel)]="modelingExercise.continuousPlagiarismControlEnabled"
+                    />
+                    <div class="plagiarism-option-label" jhiTranslate="artemisApp.exercise.continuousPlagiarismControlEnabled">Enable Continuous Plagiarism Control</div>
+                </div>
+            </div>
+        </div>
+        <div class="form-group">
```

Comment:
This config is used also for manual checks so it has to always visible

Reply:
Would it make sense to switch both around, then?
Because intuitively, if there was a checkbox and then the config related to it that it's a toggle for the config.
If there is first the config and then the checkbox, I see it as only a "refinement".

But that's just me. That's no must. Just if you can relate.

Reply:
Good point!

Reply:
Let's do this during extraction of those to a dedicated component

Reply:
Good point! Let's do this during extraction of those to a dedicated component